update/ansible/playbook/tasks/update.yml

```diff
@@ -420,7 +409,18 @@
         - ALTER TABLE api_key ALTER COLUMN name TYPE text;
       when: not ansible_check_mode
 
-    # Regenerating to enable pmm-update-perform-init
+    - name: Remove pmm-server package
```

Comment:
```suggestion
    # we need to put this step as one of the last steps, because it removes pmm.ini and /etc/alertmanager.yml
    - name: Remove old or redundant packages
```